edx_proctoring/templates/onboarding_exam/submitted.html

```diff
@@ -45,6 +45,13 @@
       </button>
     </p>
   {% endif %}
+  <p>
+    {% blocktrans %}
+      If you already have an onboarding profile approved through another course,
```

Comment:
Nit.

We could echo the language used in the onboarding panel to establish consistency.

"You may retry this exam at any time to validate that your setup still meets the requirements for proctoring."